[*] Running ...
[*] Dataset loaded: D:\ProgramLanguages\Programs\Python\code-gen\reevo/problems/tsp_gls\dataset/train200_dataset.npy with 10 instances.
D:\ProgramLanguages\Programs\Python\code-gen\reevo\problems\tsp_gls\gpt.py:30: RuntimeWarning: divide by zero encountered in divide
  
D:\ProgramLanguages\Programs\Python\code-gen\reevo\problems\tsp_gls\gpt.py:30: RuntimeWarning: invalid value encountered in divide
  
[*] Instance 0: 10.560360211540672
[*] Instance 1: 10.701890832649287
[*] Instance 2: 10.849737430346975
[*] Instance 3: 10.38442973939829
[*] Instance 4: 10.53654836982102
[*] Instance 5: 10.411002675930714
[*] Instance 6: 11.12369664933335
[*] Instance 7: 10.717109335722132
[*] Instance 8: 10.55286116452654
[*] Instance 9: 10.28321676279496
[*] Average:
10.612085317206395
